lms/templates/instructor/instructor_dashboard_2/cohorts.underscore

```diff
@@ -1,50 +1,51 @@
-<h2 class="section-title">
-    <span class="value"><%- gettext('Cohort Management') %></span>
-    <span class="description"></span>
-</h2>
-
-<div class="cohort-management-nav">
-    <h3 class="subsection-title"><%- gettext('Assign students to cohorts manually') %></h3>
-    <form action="" method="post" name="" id="cohort-management-nav-form" class="cohort-management-nav-form">
-
-        <div class="cohort-management-nav-form-select field field-select">
-            <label for="cohort-select" class="label sr"><%- gettext("Select a cohort group to manage") %></label>
-            <select class="input cohort-select" name="cohort-select" id="cohort-select"></select>
-        </div>
-
-        <div class="form-actions">
-            <button class="form-submit button action-primary action-view sr"><%- gettext('View Cohort') %></button>
-        </div>
-    </form>
-
-    <a href="" class="action-primary action-create">
-        <i class="icon fa fa-plus" aria-hidden="true"></i>
-        <%- gettext('Add Cohort') %>
-    </a>
+<div class="cohorts-state-section">
+    <label> <input type="checkbox" class="cohorts-state" value="Cohorts-State" <%- cohortsEnabled ? 'checked="checked"' : '' %> /> <%- gettext('Enable Cohorts') %></label>
 </div>
 
-<!-- Add modal -->
-<div class="cohort-management-add-form"></div>
+<% if (cohortsEnabled) { %>
+    <div class="cohort-management-nav">
+        <hr class="divider divider-lv1" />
+        <form action="" method="post" name="" id="cohort-management-nav-form" class="cohort-management-nav-form">
+
+            <div class="cohort-management-nav-form-select field field-select">
+                <label for="cohort-select" class="label sr"><%- gettext("Select a cohort group to manage") %></label>
```

Comment:
@catong good catch! This label is only 'shown' to screen readers so you can't see it. I agree that the terminology needs to be fixed. I don't know how we missed that when we updated the terminology. I think the accessibility changes must have been made after the terminology was changed and we just didn't notice it.